nuxeo-page-provider.html

```diff
@@ -341,14 +341,15 @@
               }));
               return response;
             })
-            .catch((err) => {
-              this.dispatchEvent(new CustomEvent('response', {
+            .catch((error) => {
+              this.dispatchEvent(new CustomEvent('error', {
                 bubbles: true,
                 composed: true,
                 detail: {
-                  error: err,
+                  error,
```

Comment:
Is this syntax expected? how do you access error project property when handling event details?

Reply:
I think it is. We will have something like`{ detail: { error: {...} }`

Reply:
Yes it is: https://eslint.org/docs/rules/object-shorthand